docs/pages/router/navigating-pages.mdx

```diff
@@ -72,6 +72,13 @@ To navigate, you can provide a full path (`/profile/settings`), a relative path
 | `navigate` | Navigate to the nearest route in the navigation state. `navigate` only pushes a new screen if the new route is different (not including search parameters or the hash). Otherwise, the current screen rerenders with the new parameters. If you navigate to a route that is in the history, the stack will dismiss screens to that route. |
 | `push`     | Always pushes a new route, and never pops or replaces to existing routes. You can push the current route multiple times or with new parameters.                                                                                                                                                                                           |
 | `replace`  | Remove the current route from the history and replace it with the specified URL. This is useful for redirects.                                                                                                                                                                                                                            |
+| `switch`   | Only available within a tabbed navigator. Will switch the navigator to the tab that can render the Href.                                                                                                                                                                                                                                  |
+
+### `switch`
+
+The `switch` method is only available within a tabbed navigator and is used to change the currently rendered tab. Unlike the other methods, `switch` does not guarantee that the provided href will be rendered, it changes to the tab that **could** render the href. This is useful when a tab may have internal state and you are unsure of its currently rendered screen.
```

Comment:
```suggestion
The `switch` method is only available within a tabbed navigator and is used to change the currently rendered tab. Unlike the other methods, `switch` does not guarantee that the provided href will be rendered. It changes to the tab that **could** render the href. This is useful when a tab may have an internal state and you are unsure of the currently rendered screen.
```

I think an example here can be useful on how to use this.

Also, can we also create this as a separate section as we have for "Pushing screens" and "Replacing screens" a little below on this page or vice versa (that is, add those as sub-sections to this section)? I understand that this is out of the scope of this PR, and I can follow-up on this, but want to know your opinion on whether we should keep these sections together since they relate to the parent section, "Understanding native navigation".